python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -418,6 +419,8 @@ class RandomForestRegressor(BaseRandomForestModel, RegressorMixin):
         Perform Random Forest Regression on the input data
 
         """
+        nvtx_range_push("Fit RF-Regressor @randomforestregressor.pyx")
```

Comment:
We should really add a context manager so we can do: ```
 with nvtx_range("fit"):
     .. code here
```

and not need to remember to pop (or have to manage pop correctly in the case of exceptions or early returns)

Reply:
agreed @JohnZed ✌🏾